usagov/vote-gov-drupal
# Vote-941 -> Search Vote.gov test

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-941](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-941&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This Pr will add the testing for the search vote.gov function of the site. 

Please Note.. this test will fail on the access for the `content-editor` within the teat it is written that it should not have access but currently, it does.  the work for that fix has been ticketed here with [Vote-1231](https://bixal-projects.atlassian.net/browse/VOTE-1231)

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run `backend`/`create user` tests
5. run `npm run  cy:open`
6. select `search-fuction.cy.js` and verify that all test except content editor access is passing.  This step should fail and work for this has been ticketed out and once access is created this test should pass as expected.
